### PR TITLE
cli fetch operators & validators

### PR DESCRIPTION
**Pull request by vadiminc**

### CODE REVIEW

File: incentivized_testnet/cmd.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  while (lastEpoc + epocsAmount <= toEpoc) {
+    if (epocsAmount === MAX_EPOCS_PER_REQUEST - 1 || (lastEpoc + epocsAmount >= toEpoc)) {
+      console.log(`fetching metrics for ${lastEpoc}-${lastEpoc + epocsAmount} epocs`, epocsAmount, fromEpoc, toEpoc);
+      const form = new FormData();
+      form.append('from', lastEpoc);
+      form.append('to', lastEpoc + epocsAmount);
+      form.append('keys', records.map(item => item.publicKey.replace('0x', '')).join(','));
+      let response;
+      try {
+        const { body } = await got.post('http://e2m-prater.stage.bloxinfra.com/api/validators/details', {
```
Comment (lior-blox) [inline incentivized_testnet/cmd.js:55]: remove url to param

File: incentivized_testnet/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+```sh
+npm install
+```
+
+## Fetch operators and validators to csv files
+After first run will be created `.process.cache` file which will store the last block number which was used to grab the data.
+Next runs the data will be grabed from the block which was saved in that file.
+
+```sh
+NODE_URL=https://eth-goerli.alchemyapi.io/v2/G3aG8wN9V8jKWs0NwZkWf2-img3cH4CU node cmd.js --command fetch
```
[the remark this answers is not in the document]
Comment (vadiminc) [reply incentivized_testnet/README.md]: done

File: incentivized_testnet/cmd.js (hunk 1 above)
Comment (vadiminc) [reply incentivized_testnet/cmd.js]: done

File: incentivized_testnet/cmd.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    }
+    if (Array.isArray(item.attestations)) {
+      item.attestations = item.attestations.reduce((a, b) => a + b, 0) / item.attestations.length;
+    }
+  });
+
+  stringify(records, {
+    header: true
+  }, (err, output) => {
+    fs.writeFile(`${__dirname}/validators_extra.csv`, output, () => { console.log(`exported ${records.length} validator metricks records`) });
```
[the remark this answers is not in the document]
Comment (vadiminc) [reply incentivized_testnet/cmd.js]: done

File: incentivized_testnet/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+```
+
+## Fill att and eff for validators and export to validators_extra csv file
+
+```sh
+node cmd.js --epocs 45891 45893 --command metrics
```
[the remark this answers is not in the document]
Comment (vadiminc) [reply incentivized_testnet/README.md]: done

**Pull request merged**